crates/bitwarden/src/uniffi_support.rs

```diff
@@ -18,6 +18,18 @@ uniffi::ffi_converter_forward!(
     crate::UniFfiTag
 );
 
+impl UniffiCustomTypeConverter for DecryptedString {
```

Comment:
I don't think this is needed here, we're already forwarding the definition of SensitiveString from bitwarden-crypto and DecryptedString is just a type alias right?